src/config/media/pipewire.md

```diff
@@ -26,6 +23,66 @@ directory:
 # ln -s /usr/share/applications/pipewire.desktop /etc/xdg/autostart/pipewire.desktop
 ```
 
+## Session Management
+
+PipeWire requires a session manager to function. The reference
+[`pipewire-media-session`](https://gitlab.freedesktop.org/pipewire/media-session)
+package was originally provided and, in Void, configured to run by default to
+satisfy this requirement. However, `pipewire-media-session` is deprecated and
+the authors recommend using
+[WirePlumber](https://pipewire.pages.freedesktop.org/wireplumber/) in its place.
+Install the `wireplumber` package to use this session manager with PipeWire.
+
+The standard Void configuration, which causes `pipewire` to launch
+`pipewire-media-session` automatically, must be overridden to use `wireplumber`.
+The only change necessary is to comment out the `pipewire-media-session`
+invocation in the `context.exec` section, which can be done with a single `sed`
+substitution. To make this configuration change visible to all users, place the
+new configuration file in `/etc/pipewire`:
+
+```
+# mkdir -p /etc/pipewire
+# sed '/path.*=.*pipewire-media-session/s/{/#{/' \
+    /usr/share/pipewire/pipewire.conf > /etc/pipewire/pipewire.conf
+```
+
+Alternatively, place the new configuration file in the expected location for a
+single user:
+
+```
+$ : "${XDG_CONFIG_HOME:=${HOME}/.config}"
+$ mkdir -p "${XDG_CONFIG_HOME}/pipewire"
+$ sed '/path.*=.*pipewire-media-session/s/{/#{/' \
+    /usr/share/pipewire/pipewire.conf > "${XDG_CONFIG_HOME}/pipewire/pipewire.conf"
+
+```
+
+Now, configure `wireplumber` to start in the same fashion as the `pipewire`
+executable. The `wireplumber` package includes a Desktop Entry file that can be
+linked to the Desktop Application Autostart directory:
+
+```
+# ln -s /usr/share/applications/wireplumber.desktop /etc/xdg/autostart/wireplumber.desktop
+```
+
+When launching `pipewire` and `wireplumber` from a `.xinitrc` script, it may be
+necessary to launch `wireplumber` *before* `pipewire` to prevent the latter from
+quitting if it attempts to communicate with the session manager before
+`wireplumber` launches.
+
+It is also possible to modify
+[pipewire.conf(5)](https://man.voidlinux.org/pipewire.conf.5) to automatically
+start `wireplumber`, which may make process management simpler.
+
+> In its default configuration, `wireplumber` requires an active [D-Bus
+> session](../session-management.md#d-bus). When using Desktop Application
+> Autostart symlinks, it should be sufficient to wrap the launch of a window
+> manager or desktop environment in `dbus-run-session`. When launching
+> `pipewire` and `wireplumber` from a `.xinitrc` script, it may be necessary to
+> create a new shell script suitable for passing to `dbus-run-session` that will
+> launch `pipewire`, `wireplumber` and the window manager or desktop environment
+> in a common D-Bus session.
```

Comment:
How about I strike the "when launching from .xinitrc" paragraph (to avoid endorsing that as a solution) and change the note to

> In its default configuration, `wireplumber` requires an active [D-Bus session](../session-management.md#d-bus). When using Desktop Application Autostart symlinks, it should be sufficient to wrap the launch of a window manager or desktop environment in `dbus-run-session`. Users of window managers that do not respect the autostart mechanism should consult upstream documentation regarding the process for automatically starting programs at launch.

At some point, we have to be willing to tell people that this isn't Ubuntu and they might need to do some learning on their own.